README.md

```diff
@@ -1 +1,5 @@
-# frontend
\ No newline at end of file
+# BathLARP Website Frontend

```

Comment:
Should we link to the actual website here?

Reply:
Probably not - but you have just reminded me that I planned to add the Figma link for the designs, damnit.

Reply:
We will eventually, when this is actually deployed somewhere.

Reply:
Links for the design docs added.